### ISSUE TITLE
Banner Ads seems to be hidden behind my ion-nav-view

### ISSUE THREAD

**Issue by kongwenbin** [floatinghotpot/cordova-plugin-flurry#28]
I implemented "Interstitial" and it works like a charm; using the same key, I implemented the "banner", but I don't see anything on the screen. However, when I checked on the dev dashboard, I see that there were impression counts. I am puzzled, where were the ads being displayed? 

My guess is that it might be "overlapped" by my .html ion-nav-view... is that even possible? 
Please kindly advise what can be done to resolve this issue.

What I hope to achieve here is to at least be able to see a "black patch" or "placeholder" even if there were no "banner" ads for my country.

**Comment by nuuleap** [comment]
having the same issue, in fact i can see the ad is loaded and takes up the correct space (i can see this with Chrome devtool -> inspect) but my apps view is overlayed on top of of the banner so the banner cannot be seen

**Comment by kongwenbin** [comment]
@nuuleap Yes, I checked my dev console and see that the ad has been called, and there is "impression" count. However, nothing is appearing on the screen.
